CHANGELOG.md

```diff
@@ -23,6 +23,33 @@ Main (unreleased)
 - Added `scrape_protocols` option to `prometheus.scrape`, which allows to
   control the preferred order of scrape protocols. (@thampiotr)
 
+- Updates `processor/probabilistic_sampler` to use new `FailedClosed` field from OTEL release v0.101.0
+  (but with a default value of `false`). (@StefanKurek)
+
+- Updates `receiver/vcenter` to use many new features and bugfixes introduces from OTEL releases
+  v0.100.0 and v0.101.0. Please see the release notes for more detailed descriptions here for
+  [v0.100.0](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.100.0)
+  and here [v0.101.0](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.101.0).
```

Comment:
```suggestion
- Updates `receiver/vcenter` to use new features and bugfixes introduced in OTEL releases v0.100.0 and v0.101.0.
  Refer to the [v0.100.0](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.100.0)
  and [v0.101.0](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.101.0) release
  notes for more detailed information.
```

We should avoid using `Please`. Be direct and give specific steps/instructions.

Avoid using `here` as part of an xref link (accessibility and clarity).